go/ct/gen/code_test.go

```diff
@@ -354,6 +354,142 @@ func TestCodeGenerator_ClonesCanBeUsedToResetGenerator(t *testing.T) {
 	}
 }
 
+func TestCodeGenerator_CodeIsLargeEnoughForAllConditionOps(t *testing.T) {
+
+	type constOp struct {
+		p  int
```

Comment:
Please do not use abbreviation for identifiers. It makes reading the code unnecessarily hard.